### PR TITLE
Add GOVUK styling and apply markup/class changes

### PR DESCRIPTION
**Pull request by chrishanes**
This adds the standard GOVUK styling which fixes accessibility issues around form field validation. Will need to be visually checked by a coupe of people as well as code reviewed.

### CODE REVIEW

File: app/views/booking_requests/prisoner_step.html.erb [hunk 1]
```diff
@@ -1,61 +1,59 @@
...
-      </h2>
-
-      <%= render('shared/validation', object: f.object) %>
-
+<%= form_for(@steps.fetch(:prisoner_step),
+  url: booking_requests_path,
+  html: { class: 'js-SubmitOnce', autocomplete: 'off' }) do |f| %>
+
+  <% if reviewing? %>
+  <%= render('hidden_visitors_step') %>
```
Comment (alan) [inline app/views/booking_requests/prisoner_step.html.erb]: These two lines need to be indented

File: app/views/booking_requests/prisoner_step.html.erb [hunk 2]
```diff
@@ -1,61 +1,59 @@
...
-
-        <%= single_field(f, :first_name, :text_field) %>
-
-        <%= single_field(f, :last_name, :text_field) %>
-
+        <%= single_field(f, :first_name, :text_field, class: 'form-control') %>
+        <%= single_field(f, :last_name, :text_field, class: 'form-control') %>
         <%= composite_field(f, :date_of_birth) do %>
-          <%= render('dob', form: f) %>
+        <%= render('dob', form: f) %>
```
Comment (alan) [inline app/views/booking_requests/prisoner_step.html.erb]: Same here

File: app/views/booking_requests/prisoner_step.html.erb (hunk 1 above)
Comment (chrishanes) [inline app/views/booking_requests/prisoner_step.html.erb]: Done

File: app/views/booking_requests/prisoner_step.html.erb (hunk 2 above)
Comment (chrishanes) [inline app/views/booking_requests/prisoner_step.html.erb]: Done

File: .gitignore [hunk 3]
```diff
@@ -16,3 +16,4 @@
 /coverage/*
 
 .transifexrc
+*.DS_Store
```
Comment (alan) [inline .gitignore]: Why the * ?

File: app/views/booking_requests/visitors_step.html.erb [hunk 4]
```diff
@@ -1,70 +1,62 @@
...
-      <% end %>
-
-      <h2>
+<%= form_for(@steps.fetch(:visitors_step),
+url: booking_requests_path,
+html: { class: 'js-SubmitOnce', autocomplete: 'off' }) do |f| %>
+
+<%= render('hidden_prisoner_step') %>
+<% if reviewing? %>
+<%= render('hidden_slots_step') %>
```
Comment (alan) [inline app/views/booking_requests/visitors_step.html.erb]: Needs indentation

**Comment by alan** [comment]
👍 for the code change

**Pull request merged**